devicon.json

```
@@ -8945,5 +8945,35 @@
         },
         "color": "#F04E98",
         "aliases": []
+    },
+    {
+        "name": "p5js",
+        "altnames": [
+            "p5.js"
+        ],
+        "tags": [
+           "javascript",
+           "js",
+           "library"
+        ],
+        "versions": {
+            "svg": ["original"],
+            "font": ["original"]
```

```suggestion
            "svg": [
                "original"
            ],
            "font": [
                "original"
            ]
```
Fixing entry arrays structure.